packages/components/src/promo-card/index.tsx

```diff
@@ -12,13 +12,17 @@ export default function PromoCard( { className, isMobile }: PromoCardProps ) {
 	// Using useTranslate() with interpolation to set up the title/message.
 	// https://wpcalypso.wordpress.com/devdocs/packages/i18n-calypso/README.md
 	const translate = useTranslate();
+	const redirectLink = 'https://jetpack.com/app/';
 	const components = {
-		a: <a href="https://wp.com/apps" />,
+		a: <a href={ redirectLink } />,
 	};
 	// TODO: Fix mobile logic.
 	// Should this maybe come via a media query in the CSS?
 	// Also, does it make sense to push one app store over the other?
 	// Might make more sense to have a custom "app store" SVG image that goes to wp.com/app.
+
+	// TODO: Include QR code in bundle?
```

Comment:
Including the QR SVG directly will make this component less brittle, though I could also see why we might continue serving it from s0.wp.com for convenience's sake.

Reply:
Yeah, I can see benefits to both approaches. I think I'd prefer to have the SVG data directly in the bundle, personally. I'll create a maintenance issue so we can revisit this.